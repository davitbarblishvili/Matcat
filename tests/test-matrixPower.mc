// Author: Davit
func main() int {
    matrix a;
    matrix b;
    a = [[1,2,3],[4,5,6],[7,8,9]];
    b = a^5;

    printm(b);

    return 0;

}