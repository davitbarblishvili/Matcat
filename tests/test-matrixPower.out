(
[121824.00 149688.00 177552.00 ]
[275886.00 338985.00 402084.00 ]
[429948.00 528282.00 626616.00 ]
)